Homeworks/10-my-linq/MyLinq.Tests/MyLinqTests.cs

```diff
@@ -0,0 +1,74 @@
+﻿// <copyright file="MyLinqTests.cs" company="ArtemNikit1n">
+// Copyright (c) PlaceholderCompany. All rights reserved.
+// </copyright>
+
+namespace MyLinq.Tests;
+
+#pragma warning disable SA1600
+public class MyLinqTests
+{
+    private static readonly string?[] NormalSequenceReferenceType = [null, "22", "\0", "\n"];
+    private static readonly string[]? NullSequenceReferenceType = null;
+    private static readonly int[] NormalSequence = [1, 2, 3, 4];
+    private static readonly int[] EmptySequence = [];
+
+    [Test]
+    public void Take_ShouldReturnElementsUpTo2()
+    {
+        Assert.That(NormalSequence.Take(2), Is.EqualTo(new[] { 1, 2 }));
+    }
```

Comment:
Для однострочных методов используйте лямбды
```suggestion
    public void Take_ShouldReturnElementsUpTo2() => Assert.That(NormalSequence.Take(2), Is.EqualTo(new[] { 1, 2 }));
```